cantools/database/can/message.py

```diff
@@ -27,7 +27,7 @@ def __init__(self,
                  name,
                  length,
                  signals,
-                 comment=None,
+                 comments=None,
```

Comment:
This will break scripts with `Message(..., comment="My comment.")`. I suggest that we leave the `comment` parameter unmodified and add `comments=None` at the end of the parameter list.

Reply:
okay, changed. thanks for the remark.